docs/add-a-new-service.md

```diff
@@ -51,62 +51,98 @@ If an AWS service must be created in a non-standard way, for example the service
 
 1. Add a file `internal/<service>/service_package.go` that contains an API client factory function, for example:
 
-```go
-package globalaccelerator
-
-import (
-	"context"
-
-	aws_sdkv1 "github.com/aws/aws-sdk-go/aws"
-	endpoints_sdkv1 "github.com/aws/aws-sdk-go/aws/endpoints"
-	session_sdkv1 "github.com/aws/aws-sdk-go/aws/session"
-	globalaccelerator_sdkv1 "github.com/aws/aws-sdk-go/service/globalaccelerator"
-)
-
-// NewConn returns a new AWS SDK for Go v1 client for this service package's AWS API.
-func (p *servicePackage) NewConn(ctx context.Context) (*globalaccelerator_sdkv1.GlobalAccelerator, error) {
-	sess := p.config["session"].(*session_sdkv1.Session)
-	config := &aws_sdkv1.Config{Endpoint: aws_sdkv1.String(p.config["endpoint"].(string))}
-
-	// Force "global" services to correct Regions.
-	if p.config["partition"].(string) == endpoints_sdkv1.AwsPartitionID {
-		config.Region = aws_sdkv1.String(endpoints_sdkv1.UsWest2RegionID)
-	}
-
-	return globalaccelerator_sdkv1.New(sess.Copy(config)), nil
-}
-```
+<!-- markdownlint-disable code-block-style -->
```

Comment:
I opted to disable this linter check inline rather than globally for now as the code block checking is valuable for the website docs. If this becomes too burdensome for the contributor documentation we can extend the root markdownlint configuration with a second file stored in the `docs/` subfolder and disable it globally there instead.